Luke7787/ImmaculateInventors
# Type/checking

Summary:
-add items requires folder id

How to test:
-npm run dev

![image](https://github.com/Luke7787/ImmaculateInventors/assets/121963264/c8edc1ab-17fa-4932-835e-e3d4742bfaf0)

-check that the folder id is in the item
-check that the folder has an item id in it
-you can use the already existing folder id, which should have 2 items in it already
